FaqGen/docker_compose/amd/gpu/rocm/compose.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  faqgen-tgi-service:
+    image: ghcr.io/huggingface/text-generation-inference:2.3.1-rocm
+    container_name: faggen-tgi-service
+    ports:
+      - "${FAQGEN_TGI_SERVICE_PORT}:80"
+    environment:
+      no_proxy: ${no_proxy}
+      http_proxy: ${http_proxy}
+      https_proxy: ${https_proxy}
+      TGI_LLM_ENDPOINT: "http://${HOST_IP}:${FAQGEN_TGI_SERVICE_PORT}"
+      HUGGINGFACEHUB_API_TOKEN: ${FAQGEN_HUGGINGFACEHUB_API_TOKEN}
+      HUGGING_FACE_HUB_TOKEN: ${FAQGEN_HUGGINGFACEHUB_API_TOKEN}
+    volumes:
+      - "./data:/data"
+    shm_size: 1g
+    devices:
+      - /dev/kfd:/dev/kfd
+      - /dev/dri/
+    cap_add:
+      - SYS_PTRACE
+    group_add:
+      - video
+    security_opt:
+      - seccomp:unconfined
```

Comment:
Intended for development, not production, as it disables security measures instead of adding them?

Reply:
[AMD KB](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/how-to/docker.html#docker-restrict-gpus) recommend to use this option for HPC environments to enable enables memory mapping, 
Probably will change in the future when GPU path-through method will be changed to use "--gpus" option for docker and docker compose.

Reply:
What about the PTRACE?  Surely that's needed only for debugging...

Reply:
TGI app is crashing  when trying to run rocm image without  PTRACE, so this capability is still needed, 

All docker options were taken from [PyTorch installation for ROCm manual ](https://rocm.docs.amd.com/projects/install-on-linux/en/develop/install/3rd-party/pytorch-install.html)

Reply:
> TGI app is crashing when trying to run rocm image without PTRACE, so this capability is still needed,

Ok.

That's very odd though.  Any idea why it requires that, or do you have a backtrace where it's crashing?